mslib/utils.py

```diff
@@ -32,6 +32,7 @@
 import logging
 import netCDF4 as nc
 import numpy as np
+from metpy.units import units
```

Comment:
We need to switch the UR below to the registry of metpy.

Reply:
Done